phial/bot.py

```diff
@@ -338,7 +345,7 @@ def _execute_response(self,
                                  + 'Reaction, Text')
             if response.original_ts and response.reaction:
                 self.send_reaction(response)
-            elif response.text:
+            elif response.text or response.attachments:
```

Comment:
Could you add a test case for this line. There are already some tests for this function so you should hopefully be able to just base the new test of the existing ones. The test should check that if a `Response` with attachments is passed in it call s `self.send_message`.